config/manifest.yaml

```diff
@@ -2,7 +2,7 @@ dist:
   name: grafana-ci-otelcol
   description: Grafana's OTel Collector distribution for CI/CD observability
   output_path: ./build
-  otelcol_version: 0.109.0
+  otelcol_version: 0.113.0
```

Comment:
```
73.62 Error: invalid configuration: `otelcol_version` has been removed. To build with an older Collector API, use an older (aligned) builder version instead
```

Build is failing https://github.com/grafana/grafana-ci-otel-collector/actions/runs/11741755194/job/32711236526?pr=167 🙁